cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js

```diff
@@ -0,0 +1,148 @@
+          testData.instance = instanceData;
+
+          Organizations.createOrganizationViaApi(testData.organization);
+        });
+      })
+      .then(() => {
+        ServicePoints.createViaApi(testData.servicePoint).then(() => {
+          testData.location = Locations.getDefaultLocation({
+            servicePointId: testData.servicePoint.id,
+          });
+
+          Locations.createViaApi(testData.location).then(() => {
+            Orders.createOrderViaApi(NewOrder.getDefaultOrder(testData.organization.id)).then(
+              (number) => {
+                testData.orderNumber = number;
+
+                InventoryHoldings.getHoldingsFolioSource().then((folioSource) => {
+                  InventoryInstances.createHoldingViaAPI({
+                    instanceId: testData.instance.instanceId,
+                    permanentLocationId: testData.location.id,
+                    sourceId: folioSource.id,
+                  }).then(({ id: holdingId }) => {
+                    testData.instance.holdingId = holdingId;
+                  });
+                });
+              },
+            );
+          });
+        });
+      });
+
+    cy.createTempUser([
+      // extra perms
+      Permissions.uiInventoryViewCreateEditInstances.gui,
+      Permissions.uiInventoryViewCreateEditItems.gui,
+
+      // perms from test
+      Permissions.uiInventoryViewInstances.gui,
+      Permissions.uiOrdersApprovePurchaseOrders.gui,
+      Permissions.uiOrdersEdit.gui,
+    ]).then((userProperties) => {
+      testData.user = userProperties;
+
+      cy.loginAsAdmin({
+        path: TopMenu.ordersPath,
+        waiter: Orders.waitLoading,
+      });
+      // cy.login(userProperties.username, userProperties.password, {
+      //   path: TopMenu.ordersPath,
+      //   waiter: Orders.waitLoading,
+      // });
+    });
+  });
+
+  after('Delete test data', () => {
+    InventoryHoldings.deleteHoldingRecordViaApi(testData.instance.holdingId);
+    Orders.deleteOrderByOrderNumberViaApi(testData.orderNumber);
+    InventoryInstance.deleteInstanceViaApi(testData.instance.instanceId);
+    Organizations.deleteOrganizationViaApi(testData.organization.id);
+    Locations.deleteViaApi(testData.location);
+    ServicePoints.deleteViaApi(testData.servicePoint.id);
+    Users.deleteViaApi(testData.user.userId);
+  });
+
+  it(
+    'C375232 Create new holdings for already existing location when creating an order line (thunderjet)',
+    { tags: [TestTypes.extendedPath, DevTeams.thunderjet] },
+    () => {
+      const pol = {
+        title: testData.instance.instanceTitle,
+        acquisitionMethod: ACQUISITION_METHOD_NAMES_IN_PROFILE.PURCHASE_AT_VENDOR_SYSTEM,
+        orderFormat: ORDER_FORMAT_NAMES.PHYSICAL_RESOURCE,
+        quantity: '1',
+        price: '20',
+        materialType: MATERIAL_TYPE_NAMES.BOOK,
+        inventory: 'Instance, holdings, item',
+      };
+
+      // Click on "PO number" link on "Orders" pane
```

Comment:
I think that comment doesn't need here

Reply:
The comment left to make it easier to understand what test is actually doing. It's a step from related test case

Reply:
I think the test should be written in such a way that you can understand what's going on here without comment.
The best way to use comments is to use them for some part of the methods.